src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/gateways/PrisonApiGateway.kt

```diff
@@ -43,4 +44,15 @@ class PrisonApiGateway(private val prisonApiWithAuthWebClient: WebClient) {
       .bodyToMono(String::class.java)
       .block()
   }
+
+  fun getOffenderBooking(bookingId: Long): OffenderBookingResponse = prisonApiWithAuthWebClient
+    .get()
+    .uri("/api/booings/$bookingId")
```

Comment:
didn't realise we were `booing` in Prison API these days :joy:

Reply:
haha